src/style.css

```
@@ -0,0 +1,134 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  box-sizing: border-box;
+  --background-color: #fff;
+  --second-background-color: #00acdc;
+  --text-color: #060b35;
+  --second-text-color: #616070;
+}
+
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: var(--background-color);
+  border: 1px solid #f3f3f3;
```

create a variables for colors
```suggestion
  border: 1px solid $c-color;
```